core/primitives/src/views.rs

```diff
@@ -394,6 +394,15 @@ pub struct EpochInfoView {
     pub validators: Vec<ValidatorInfo>,
 }
 
+#[cfg_attr(feature = "deepsize_feature", derive(deepsize::DeepSizeOf))]
+#[derive(Serialize, Deserialize, Debug)]
+pub struct ChunkInfoView {
+    pub num_of_blocks_in_progress: usize,
+    pub num_of_chunks_in_progress: usize,
+    pub num_of_orphans: usize,
+    pub next_blocks_log: Vec<String>,
```

Comment:
comment what's inside this field

Reply:
Going to split this one into multiple columns.